src/index.sass

```diff
@@ -8,9 +8,14 @@ $primary-color: #4882ff
     box-sizing: border-box
     margin: 0
     padding: 0
+*:before,
+*:after 
+    box-sizing: border-box;
 
 html
     -webkit-text-size-adjust: none
+    height: 100%
```

Comment:
Inconsistencies with ";" and no ";". They are in some other places in your new additions as well that you could change.

Reply:
It doesn't matter if it's with ; or without, the important thing is that it's consistent.